openmicroscopy/openmicroscopy
# Load thumbnails

manual rebase of https://github.com/openmicroscopy/openmicroscopy/pull/5062

--------------

# What this PR does

- https://github.com/openmicroscopy/openmicroscopy/pull/5081 fix getThumbnailsByLongestSideSet in readonly group https://trello.com/c/3eEBEPG4/7-retrieving-multiple-thumbnails 
- allow loading thumbnails in a batches

# Testing this PR

- test thumbnail version like in https://github.com/openmicroscopy/openmicroscopy/pull/3713#issuecomment-95146309
- load plate, all thumbnail should be loaded in a batches
- log in and test if updating rendeting update thumbnail:
  - test centre panel, search and plate grid
- consider any usecases I could miss! :)

- [ ] **performance testing** That should happen on real production deployment (with and without nginx micro cache turned on)

cc @pwalczysko 

--thumbnails
--exclude
